docker_test.go

```diff
@@ -190,7 +191,9 @@ func TestContainerWithHostNetworkOptions_UseExposePortsFromImageConfigs(t *testi
 		t.Fatal(err)
 	}
 
-	defer nginxC.Terminate(ctx)
+	defer func() {
```

Comment:
For the deferred functions and container termination, could you apply this pattern? https://github.com/testcontainers/testcontainers-go/pull/552/files#diff-037b83426e676aa3786c08b334b0db7bb70235792961cbe4ed3340bc0a60a7a8R2244